spikingjelly/activation_based/distributed/vision/config.py

```
@@ -0,0 +1,352 @@
+        builder type metadata.
+
+        :return: 可序列化配置。 / Serializable configuration.
+        :rtype: dict[str, Any]
+        """
+        return _encode(self)
+
+    @classmethod
+    def from_dict(cls, data: dict[str, Any]) -> TrainingConfig:
+        r"""Restore a configuration created by :meth:`as_dict`.
+
+        **中文：** 从 ``as_dict`` 结果恢复具体 config 子类。自定义 config 类必须先
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)